packages/core/state/index.ts

```diff
@@ -8,6 +8,7 @@ import { PersistedConfig, PersistedConfigKeys } from "../services/PersistentConf
 import interaction, { InteractionStateBranch } from "./interaction";
 import metadata, { MetadataStateBranch } from "./metadata";
 import selection, { SelectionStateBranch } from "./selection";
+import Annotation from "../entity/Annotation";
```

Comment:
This import belongs with the above grouping (so as a new line above line 6)

Reply:
Moving this import causes the app to go blank. I am completely lost on why.

Reply:
Pulled locally and tried this as well, also caused a load error for me when put anywhere from line 8 up:
`Uncaught ReferenceError: Cannot access 'Annotation' before initialization`